bluetooth-plane/index.html

```diff
@@ -0,0 +1,207 @@
+      }
+      paper-slider {
+        width: 100%;
+      }
+      paper-slider.blue {
+        paper-slider-active-color: var(--paper-light-blue-500);
+        paper-slider-knob-color: var(--paper-light-blue-500);
+      }
+      paper-button {
+        display: block;
+        margin-bottom: 24px;
+      }
+      paper-button.colorful {
+        color: #4285f4;
+      }
+      paper-button[raised].colorful {
+        background: #4285f4;
+        color: #fff;
+      }
+      paper-button[toggles] {
+        transition: background-color 0.3s;
+      }
+      paper-button[toggles][active] {
+        background-color: rgba(0, 0, 0, 0.25);
+      }
+      paper-button[toggles][active].colorful {
+        background-color: rgba(66, 133, 244, 0.25);
+      }
+      paper-button[toggles][active][raised].colorful {
+        background-color: rgba(66, 133, 244, 0.75);
+      }
+      paper-button.blue {
+        color: var(--paper-light-blue-500);
+        paper-button-flat-focus-color: var(--paper-light-blue-50);
+      }
+      body {
+        background-color: var(--paper-grey-50);
+      }
+      #cards {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 400px;
+      }
+      paper-card {
+        margin-bottom: 16px;
+        width: 100%;
+      }
+    </style>
+  </head>
+  <body unresolved>
+    <div id="cards">
+      <paper-card heading="Bluetooth Toy Plane">
+        <div class="card-content">
+          <paper-progress id="progress" indeterminate></paper-progress>
+        </div>
+      </paper-card>
+
+      <paper-card>
+        <div class="card-content">
+          <paper-button id="left" raised class="colorful">Left</paper-button>
+          <paper-button id="center" raised class="colorful">Center</paper-button>
+          <paper-button id="right" raised class="colorful">Right</paper-button>
+          <paper-button id="stop" raised class="colorful">Stop</paper-button>
+        </div>
+      </paper-card>
+
+      <paper-card>
+        <div class="card-content">
+          <div class="center horizontal layout">
+            <div>Engine</div>
+            <paper-slider id="engine" class="red" value="0" max="254" editable></paper-slider>
+          </div>
+        </div>
+      </paper-card>
+
+      <paper-card>
+        <div class="card-content">
+          <div class="center horizontal layout">
+            <div>Rudder</div>
+            <paper-slider id="rudder" class="red" value="0" min="-128" max="127" editable></paper-slider>
+          </div>
+        </div>
+      </paper-card>
+
+      <paper-dialog id="dialog">
+        <h2>Error</h2>
+        <p>Could not connect to bluetooth device!</p>
+      </paper-dialog>
+    </div>
+
+    <platinum-bluetooth-device services-filter='["f000ffc0-0451-4000-b000-000000000000"]' id="plane-device">
+      <platinum-bluetooth-characteristic id="engineCharacteristic"
+        service="75b64e51-f181-4ed1-921a-476090d80ba7"
```

Comment:
I just filed an issue to keep track of this and see how developers would like it: https://github.com/PolymerElements/platinum-bluetooth/issues/6